surveys/templates/survey_publish.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+  <h2>Publish survey</h2>
+  <form method="POST" class="post-form">{% csrf_token %}
+     <div>Are you sure you want to publish <b>{{survey.name}}</b>?</div>
```

Comment:
Any reason for these to be `<div>`s instead of `<p>`s? (And for them to be nested inside the form?)

Reply:
None of the forms have final styles, I figured that should be a separate issue!

Reply:
I can switch these to `<p>`s though